### ISSUE TITLE
Document New Routes.js <Set />: add instructions and update Router Doc

### ISSUE THREAD

**Issue by thedavidprice** [redwoodjs/redwoodjs.com#638]
@Tobbe @jtoar @cannikin 

Reference https://github.com/redwoodjs/redwood/pull/1898

Some next steps:
- [x] Update https://github.com/redwoodjs/redwoodjs.com/blob/main/docs/router.md
- [x] Create Forum Topic with "how to" example -- link to it from Release Notes
- [x] Discussion: Do we add this to the Tutorial? Or maybe create a Cookbook article instead?

**Comment by Tobbe** [comment]
Thanks for writing this issue David! I talked briefly with Dom about this, and we both think we should update the tutorial to use `<Set>`. I said I could do that part. Dom agreed to help out with /docs/router.md. We didn't talk about a forum post, but I'll coordinate with him about that.

Unless I hear anything else, I'll move forward with the tutorial updates. Then we'll just leave that PR open until we've decided if that's what we want to do or not.
